src/Features/ERDDAP/waterLevel/chart/chartDisplay.tsx

```diff
@@ -49,14 +51,21 @@ export const WaterLevelChartDisplay: React.FunctionComponent<ChartTimeSeriesDisp
           acc[level.name] =
             level.name === "Major"
               ? {
-                  minValue: dataConverter.convertToNumber(level.min_value, unitSystem) + datumOffset,
-                  maxValue: dataConverter.convertToNumber(level.min_value, unitSystem) + 1 + datumOffset,
+                  minValue:
+                    dataConverter.convertToNumber(level.min_value, unitSystem) -
```

Comment:
It might make sense to swap the order of things here, and maybe capture the datum offsetting in it's own function so that we can test it (map MHHW it over an array, ask for the max and see that it's near 0, map MLLW over an array, get the min, see that it's near 0). Then apply the converter to the output of the already offset value.